src/libs/actions/Policy/Policy.ts

```diff
@@ -1259,7 +1259,7 @@ function updateGeneralSettings(policyID: string, name: string, currencyValue?: s
         (request) => request.data?.policyID === policyID && request.command === WRITE_COMMANDS.CREATE_WORKSPACE && request.data?.policyName !== name,
     );
 
-    const createWorkspaceRequest = persistedRequests[createWorkspaceRequestChangedIndex];
+    const createWorkspaceRequest = persistedRequests.at(createWorkspaceRequestChangedIndex);
```

Comment:
Since `createWorkspaceRequestChangedIndex` can be -1 make sure the logic won't affected.